snap/snapcraft.yaml

```diff
@@ -0,0 +1,35 @@
+name: hwctl
+base: core22
+version: '0.1' # rewritten with the help of `adopt-info` below
```

Comment:
right now the version (at least for the deb package and the lib) is `0.0.1`. should we keep it the same here?

Reply:
The version to all artifacts should be coming from a git tag. I've fixed this up now for the snap build on this branch, can advise how to do that with the deb build separately.